src/components/Account.vue

```diff
@@ -38,55 +50,145 @@
         @Emit()
         // tslint:disable-next-line no-empty
         private changed(label: string) {}
+
+        @Watch('image', { immediate: true })
+        private onImageChange() {
+            this.showImage = !!this.image;
+
+            // load clip path if not done so yet
+            if (!this.showImage || document.getElementById('nimiq-hexagon-clip')) return;
+            /* tslint:disable max-line-length */
+            document.body.insertAdjacentHTML('beforeend', `
```

Comment:
Will this add and add and add the svg inline whenever the image prop changes?

Reply:
No, see line above